common/src/main/scala/services/teamwork/SquadAction.scala

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2019 PSForever
+package services.teamwork
+
+import net.psforever.objects.zones.Zone
+import net.psforever.packet.game._
+import net.psforever.types.{PlanetSideEmpire, SquadRequestType, Vector3}
+
+object SquadAction {
+  trait Action
```

Comment:
Please seal all traits that wont be extended to improve compiler type checking and to generate `Warning:(7, 3) match may not be exhaustive.` on partial matches
https://www.waitingforcode.com/scala-core/sealed-keyword-scala/read

Reply:
Maybe we can seal them by putting classes into files with other classes?  I was using `FooAction.Action` and `FooResponse.Response` as `FooServiceMessage` and `FooServiceResponse` message parameter types, respectively.  While I sometimes might feel that this is not a good thing (for one reason or another), I believe that's why I chose to not seal them as-is.

Reply:
Sealed.